package nextstep.utils.exception;

public class ClassToInstanceFailedException extends RuntimeException {
    public ClassToInstanceFailedException(Throwable cause) {
        super(cause);
    }
}
